com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -567,13 +565,28 @@ half3 VertexLighting(float3 positionWS, half3 normalWS)
 half4 UniversalFragmentPBR(InputData inputData, half3 albedo, half metallic, half3 specular,
     half smoothness, half occlusion, half3 emission, half alpha)
 {
+    // Decreases smoothness as the directionality of the light decreases. This approximates the specular highlight
+    // spreading and scattering, as the light becomes less directional. Without this, even shadowed areas look shiny.
+    // This is recommended by: https://media.contentapi.ea.com/content/dam/eacom/frostbite/files/gdc2018-precomputedgiobalilluminationinfrostbite.pdf
+    // Although, here we do not apply the sqrt to the falloff. Instead we square it, which seems to produce a closer image to the blender groundtruth.
+    // The range remap makes sure that surfaces in perfectly direct light (directionality > .9) remain their true specular (directionality rarely reaches a perfect 1).
+
+    // TODO(john): We could collapse this math down a lot, if we knew the proper transformations from perceptual
+    // smoothness to linear smoothness. Worth optimizing if we become ALU bound. Just make sure to do a before/after.
+    // This smoothness falloff has been carefully tested to look good.
+    half linearSmoothness = 1 - PerceptualSmoothnessToRoughness(smoothness);
+    half directionality = length(inputData.bakedGI_directionWS);
+    half adjustedSmoothness = linearSmoothness * pow(RangeRemap(0.0, .9, directionality), 2);
```

Comment:
I'll give this a shot. Being able to set the directionality cutoff at .9 is very useful, because that's the value that most 'strong directional light' in the scene has after baking (from experiments). So the cutoff guarantees all surfaces hit by direct light maintain the same smoothness value. I think I can just square it though.. so .9*.9.

Reply:
Also, definitely we should over tune the visuals for this scene specifically, haha.